src/strategies/radicle-funding-nft/examples.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "radicle-funding-nft",
+      "params": {
+        "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
```

Comment:
These params aren't used, there should be a way to define the project NFT address, currently the strategy return funding amount for all projects, the strategy should give voting power for a specific project / NFT.

Reply:
do we need to query also for a specific NFT, or only adding the capability to query for a `fundingProject` is required?

Reply:
Only `fundingProject`

Reply:
I was receiving an empty `fundingProject` field for `nfts` query.
so for finding `nfts` of a certain `fundingProject` i added another query parameters to query `fundingProjects` (but only one subgraph query is ever requested).